### PR TITLE
[FIX] Argument auto_commit in send_mail

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
New argument in method `send_mail(self, auto_commit=False)`.

### CODE REVIEW

**Comment by ThomasBinsfeld** [comment]
@adrienpeiffer @lmignon

**Comment by adrienpeiffer** [comment]
:+1:

File: mail_optional_autofollow/wizard/mail_compose_message.py [hunk 1]
```diff
@@ -22,8 +22,9 @@ def default_get(self, fields_list):
...
 
     @api.multi
-    def send_mail(self):
+    def send_mail(self, auto_commit=False):
         for wizard in self:
             super(MailComposeMessage, wizard.with_context(
-                mail_post_autofollow=wizard.autofollow_recipients)).send_mail()
+                mail_post_autofollow=wizard.autofollow_recipients)).send_mail(
+                    auto_commit=auto_commit)
         return {'type': 'ir.actions.act_window_close'}
```
Comment (pedrobaeza) [inline mail_optional_autofollow/wizard/mail_compose_message.py]: As you are touching here, you can simply `return True`

**Comment by pedrobaeza** [comment]
:+1: squashing on merge.

**Comment by ThomasBinsfeld** [comment]
@pedrobaeza Squashed

**Pull request merged**